### DISCUSSION TITLE
Open outlook and a specific folder from an hyperlink command (W10 environment)

### DISCUSSION THREAD

**Discussion by paulhux** [freeplane/freeplane#1880] [Help] [answered]
In the command prompt of Windows10, `outlook.exe /select "outlook:\\myname@mydomain\Draft"` launches outlook and opens the Draft folder.

I try to do the same from Freeplane, namely, launch outlook and open a specified folder with Insert -> Add hyperlink to execution of an OS command.

I managed to execute explorer or notepad from a node with  `explorer` and `notepad` hyperlinks.

I can't execute Outlook with `outlook`, `outlook.exe`, `C:\ProgramFiles (x86)\Microsoft Office\Office16\OUTLOOK.EXE` or `cmd C:\ProgramFiles (x86)\Microsoft Office\Office16\OUTLOOK.EXE` hyperlinks.

Someone has a clue ?

**Comment by macmarrum** [comment, marked as answer]
I think the path to outlook.exe needs to be in quotes, because it contains spaces.

I've seen Outlook installed in `C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE`, and the folder being named `Drafts`. For this location and name, the command would be `"C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE" /select "outlook:\\myname@mydomain\Drafts"`

**Comment by paulhux** [comment]
Holy grail, how could I be so dumb.
It works fine. You're great.
